Flabbergasted-Flamingos/Bangazon-admin-app
# added where you can't edit trainings that have started

# Description
Changed the training_detailed view so you can't edit a training that has started


## Steps to Test Solution

1. git fetch --all
2. git checkout JCtrainingdate
3. check your trainings and see if you can edit past dates
4. Create a future training and see if you are able to edit it
